### ISSUE TITLE
About the code

### ISSUE THREAD

**Issue by CXiaoDing** [sailordiary/LipNet-PyTorch#2]
@sailordiary Hi, thank you for your code!
I want to run it on my computer, but I don't know how to start my work.
Can you tell me something about running this code? For example, what structure should the dataset be and where should I put the datasets and pretrained weight?
Thank you very much!
Can I add your wechat or QQ if you are convenient？

**Comment by sailordiary** [comment]
Codebase migrated to PyTorch 1.x and refactored, should be clearer now :)

**Comment by kunshou123** [comment]
@sailordiary  hi, thank you for your code!
 i want to know whether the model have a quick convergence speed? i use Dense3D model to train but find  loss cannot quickly decrease .Do you know some good lipreading models  spend little time on training,thank you
